tests/thread/enc/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What happened here? What was the reasoning behind moving the build and sources for `pthread_enc` and `oethread_enc`?

Reply:
As of now since we are not passing in subdirectories, in add_enclave_test I have enclave path set to "enc" and host path set to "host" so Windows can figure out where the Linux binaries are. In the thread case since the enclave path was different from "enc" it lead to tests breaking. Hence the requirement to move them to "enc".